arduino/sketch/sketch.go

```diff
@@ -64,6 +64,11 @@ func New(path *paths.Path) (*Sketch, error) {
 	}
 
 	path = path.Canonical()
+	if exist, err := path.ExistCheck(); err != nil {
+		return nil, fmt.Errorf("%s: %s", tr("sketch path is not valid"), err)
```

Comment:
maybe it would be cool to have a test also for this?

Reply:
It's a bit hard test this branch, under the hood `ExistCheck` does a `Stat` call, but I don't know how to make it fail... if you have any idea... :-)

Reply:
Mhh you are right.. It seems difficult